src/csharp/Grpc.Core.Api/SerializationContext.cs

```diff
@@ -27,13 +28,31 @@ public abstract class SerializationContext
     {
         /// <summary>
         /// Use the byte array as serialized form of current message and mark serialization process as complete.
-        /// Complete() can only be called once. By calling this method the caller gives up the ownership of the
+        /// <c>Complete()</c> can only be called once. By calling this method the caller gives up the ownership of the
         /// payload which must not be accessed afterwards.
         /// </summary>
         /// <param name="payload">the serialized form of current message</param>
         public virtual void Complete(byte[] payload)
         {
             throw new NotImplementedException();
         }
+
+        /// <summary>
+        /// Gets buffer writer that can be used to write the serialized data. Once serialization is finished,
+        /// <c>Complete()</c> needs to be called.
+        /// </summary>
+        public virtual IBufferWriter<byte> GetBufferWriter()
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Complete the payload written to the buffer writer. Complete() can only be called once.
```

Comment:
nit for consistency with the doc above:
```suggestion
        /// Complete the payload written to the buffer writer. <c>Complete()</c> can only be called once.
```